File: 001-BitwiseOperators/014.rightmost-different-bit.js
const posOfRightMostDiffBit = (m, n) => {
  let x = m ^ n;
  let ans = 1;
  while (x) {
    if (x % 2 == 1) {
      return ans;
    }
    x = parseInt(x / 2);
    ans++;
  }
  return -1;
};

File: GeeksForGeeks/DS-Course/001-BitwiseOperators/014.rightmost-different-bit.py
class Solution:
    
    #Function to find the first position with different bits.
    def posOfRightMostDiffBit(self,m,n):
        x = m ^ n
        ans = 1
        while x:
            if x % 2 == 1:
                return ans
            x = x // 2
            ans += 1
        
        return -1